app/context_manager.py

```diff
@@ -0,0 +1,23 @@
+from os import remove, makedirs, path
+
+
+class MoveFileManager:
+    def __init__(
+            self,
+            source_file_name: str,
+            path: str
+    ) -> None:
+        self.source_file_name = source_file_name
+        self.path = path
+
+    def __enter__(self) -> None:
+        if self.path and not path.exists(self.path):
+            makedirs(self.path)
+
+    def __exit__(
+            self,
+            exc_type: str,
+            exc_val: str,
+            exc_tb: str

```

Comment:
The type annotations for `exc_type`, `exc_val`, and `exc_tb` in the `__exit__` method are incorrect. According to Python's context manager protocol, `exc_type` should be of type `Optional[Type[BaseException]]`, `exc_val` should be `Optional[BaseException]`, and `exc_tb` should be `Optional[TracebackType]`. Consider updating these annotations to reflect the correct types.